datasets/ps/local_freezing/ps_local_freezing.yaml

```diff
@@ -0,0 +1,26 @@
+title: Palestine Monetary Authority Local Freezing List
+entry_point: crawler.py
+prefix: ps-freezing
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  People who are sanctioned by the PMA in accordance with Security Council Resolutions to combat money laundering and the financing of terrorism .
+description: |
+  The Palestine Monetary Authority (PMA) is a public institution
+  responsible for the formulation and implementation of monetary
+  and banking policies in Palestine.  The PMA operates by an act 
+  f the Palestine Legislative Council's PMA Law Number (2) of 1997,
+  which stipulated the proper authority and autonomy of the PMA and the Banking Law Number 9 of 2010.
+  In Line with authorities of the "Security Council Resolutions Enforcing Committee",
+  which formed according to Decree No. (14) of 2015 Concerning the Enforcement of
+  Security Council Resolutions, the PMA publishes a Local Freezing List.
```

Comment:
Nice. Could you expand a bit on what it means to be on the freezing list? 

A quick google for the resolution gave me a few UK and UN links, as well as this https://www.ffu.ps/files/2021/high/032021/3.%20Enforcement%20of%20Security%20Council%20Resolutions.pdf which we could quote and link to, if it's the right one.